addon/components/single-event-objective-list.hbs

```diff
@@ -15,42 +28,45 @@
       </button>
     {{/if}}
   </h2>
-  {{#if this.groupByCompetencies}}
-    {{#if this.domains.length}}
-      <ul class="tree">
-        {{#each this.domains as |domain|}}
-          <li>
-            {{domain.title}}
-            <ul>
-              {{#each domain.objectives as |title|}}
-                {{! template-lint-disable no-triple-curlies }}
-                <li>
-                  {{{title}}}
-                </li>
-              {{/each}}
-            </ul>
+  {{#if this.isExpanded}}
+    {{#if this.groupByCompetencies}}
+      {{#if this.domains.length}}
+        <ul class="tree" data-test-tree>
+          {{#each this.domains as |domain|}}
+            <li data-test-domain>
+              <span data-test-domain-title>{{domain.title}}</span>
+              <ul>
+                {{#each domain.objectives as |title|}}
+                  {{! template-lint-disable no-triple-curlies }}
```

Comment:
I'm wondering in light of https://github.com/ilios/ilios/issues/3571 if we should create an ObjectiveTitle component that handles this and consolidates the no-tripple-currlies as well.

Reply:
maybe. seems a bit excessive to create a component wrapper just to render triple curlies. but maybe i'm not understanding the full scope of the issue here.

Reply:
follow-up item for sure.